osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -248,6 +251,9 @@ private void updateNestedPositions()
 
             if (TailCircle != null)
                 TailCircle.Position = EndPosition;
+
+            if (LastRepeat != null)
+                LastRepeat.Position = RepeatCount % 2 == 0 ? Position : Position + Path.PositionAt(1);
```

Comment:
My question at this point would be - why does only the last repeat get this treatment? Surely all of them should?